index.html

```diff
@@ -124,6 +124,89 @@ <h2><dfn>CropTarget Production</dfn></h2>
+            </li>
+            <li>
+              <var>T</var>.<a data-cite="mediacapture-streams/#dfn-readystate">readyState</a> is
+              <a data-cite="mediacapture-streams/#idl-def-MediaStreamTrackState.live">"live"</a>.
+            </li>
+            <li><var>T</var> is not <a data-cite="mediacapture-region#dfn-cropped">cropped</a>.</li>
+          </ul>
+        </section>
+        <section>
+          <h4>Elements eligible for restriction</h4>
+          <p>
+            We say that an {{Element}} <var>E</var> is <dfn>eligible for restriction</dfn> if and
+            only if it fulfils all of the following conditions:
+          </p>
+          <ul>
+            <li><var>E</var> forms a stacking context.</li>
+            <li>
+              <var>E</var> is
+              <a data-cite="css-transforms-2#grouping-property-values">flattened in 3D.</a>
+            </li>
+            <li>
+              <var>E</var> forms a <a data-cite="filter-effects-2/#backdrop-root">backdrop root</a>.
+            </li>
+          </ul>
+        </section>
+        <h4>Valid restriction targets</h4>
+        <p>
+          We say that an {{Element}} <var>E</var> is a <dfn>valid restriction target</dfn> for a
+          {{MediaStreamTrack}} <var>T</var>, if and only if all of the following conditions hold:
+        </p>
+        <ul>
+          <li><var>T</var> is a [=restrictable MediaStreamTrack=].</li>
+          <li><var>E</var> is a [=eligible for restriction=].</li>
+          <li>
+            The [=top-level browsing context=] of the
+            <a data-cite="screen-capture/#dfn-display-surface">display surface</a>
+            that is the source of <var>T</var>, is <var>E</var>'s [=shadow-including root=].
+          </li>
+        </ul>
+        <div class="note">
+          <p>
+            Informally, this means that <var>T</var> is an active video track associated with
+            tab-capture, and <var>E</var> is an Element [=connected=] to the DOM in the captured
+            tab.
+          </p>
+          <p>
+            Note that whether an Element <var>E</var> is a [=valid restriction target=] for a
+            {{MediaStreamTrack}} <var>T</var> may change either before or after a capture starts, as
+            well as before or after restriction starts. Examples include:
+          </p>
+          <ul>
+            <li><var>T</var> is stopped programmatically.</li>
+            <li><var>T</var> is stopped by the user.</li>
+            <li>
+              <var>T</var>.<a data-cite="mediacapture-streams/#dfn-source-0">[[\Source]]</a> changes
+              due to user interaction with the user agent and/or operating system.
+            </li>
+            <li><var>E</var>'s set of CSS attributes changes.</li>
+          </ul>
+          <p>Invalidity before restriction starts will suppress restriction.</p>
+          <p>
+            Invalidity after restriction starts will suppress additional frames until validity is
```

Comment:
Yes, I believe this has been discussed before, although I don't presently recall where. The TL;DR of my answer is that it's not necessary for the MVP, but it can be added later if developers require it. (I challenge you, in a most friendly manner, to come up with a non-overly-contrived example of the event being actionable and non-trivial and an application being likely to register an event handler for it. If the action that causes muting is taken by the app, the event is not necessary; if by an embedded app, they're collaborating to begin with and can exchange message; if by the crashing of the embedded app or its navigation, some use could be derived of the event, I concede. I cannot think of other examples.)